### DISCUSSION TITLE
Alternate solution for And Operator.

### DISCUSSION THREAD

**Discussion by atindersingh67** [appbaseio/reactivesearch#2038] [Q&A]
I am using Data Search in my react app. 
`      <DataSearch
                        componentId="result"
                        dataField = {[
                          "categoryname",  
                          "description"
                        ]
                        fieldWeights = { [5, 5] }
                        fuzziness = {"AUTO"}
                        autosuggest={true}
                        queryFormat="and"
                        URLParams={true}
                    />`

When user is trying to search for something , for eg. 'Digital Custom Art', Its not return any result, Because in DB there is no product with all three keyword.
But there are products that have 'Custom Art' in their description. Is there any way those can be picked? 

I tried using ` queryFormat="or"` But with this its first showing products having  'Custom' in the description then products with 'Art'.

Is there any way i can get my desired result.

**Comment by siddharthlatest** [comment]
@atindersingh67 This looks like something you can control with fieldWeights. Assign a higher weight to `categoryname` field than `description` if you want the results that have values from that field to show up at the top. queryFormat `or` can work well here, as it will sort the results that contain all or most terms, based on the search relevance.
